src/lib/mongoBackend/mongoRegistrationsGet.cpp

```diff
@@ -0,0 +1,317 @@
+/*
```

Comment:
All the set*() functions are already in mongoRegistrationGet.cpp (without 's'). 

If the approach done for subscriptions is not suitable (both mongo methods defined in mongoGetsubscriptions.h/cpp along with all the common functions needed by them) them having a library of common methods should be done.

Reply:
Well, all those functions hardcode the mongo field name inside.
That's why I decided to not use them.

However, it's easy enough to remove the field name and have it as a parameter.

Reply:
Fixed in 72f4e9f

Reply:
Maybe I'm not made my point clear... 

I'm not meaning reusing the set* methods used by get subscription(s) logic in the get registration(s) logic. What I mean is to use in this PR the set* methods already defined for get registration (without 's'). Note that the db field is the same in both cases.

In other words, ideadlly this PR shouldn't include any new method as the needed ones are already there.

Look to the subscription code. The idea is the same. Both get subscription and get subscriptions are using the same set of methods to fill the response object from DB data.

If the thing is not clear after this comment, the let's move to skype when we match together.

Reply:
NTC (discussion regarding this follows in other comments)

Reply:
"Fixed" in 33bfe335a